### ISSUE TITLE
Cannot add an account

### ISSUE THREAD

**Issue by daenney** [eul-im/eul#56]
Eul.im: v0.26
Platform: Linux 4.13.11 with GNOME Shell 3.26.2

When I try to add a new account (for example by clicking on the Slack logo) nothing happens. No new window shows up to do an authentication flow etc. The same problem occurs with Skype or GMail.

I tried figuring out if there's a `debug` or `verbose` flag that I can pass to the CLI to figure out what's up but those don't seem to exist. The only other thing I managed to find in `journalctl` is this:

```
Nov 08 11:14:03 monoceros org.gnome.Shell.desktop[753]: Window manager warning: Buggy client sent a _NET_ACTIVE_WINDOW message with a timestamp of 0 for 0x2200006 ()
Nov 08 11:14:03 monoceros org.gnome.Shell.desktop[753]: Window manager warning: Buggy client sent a _NET_ACTIVE_WINDOW message with a timestamp of 0 for 0x220001a (eul.im v0.)
Nov 08 11:14:03 monoceros org.gnome.Shell.desktop[753]: Window manager warning: Buggy client sent a _NET_ACTIVE_WINDOW message with a timestamp of 0 for 0x2200021 (Add an acc)
Nov 08 11:14:06 monoceros org.gnome.Shell.desktop[753]: Window manager warning: Buggy client sent a _NET_ACTIVE_WINDOW message with a timestamp of 0 for 0x220001a (eul.im v0.)
```

**Comment by daenney** [comment]
Actually, I took another look, saw this in the journal now:

```
nov 18 00:07:15 monoceros systemd-coredump[5161]: Process 5158 (minimal) of user 1000 dumped core.
                                                  
                                                  Stack trace of thread 5158:
                                                  #0  0x00007fb9bd9838a0 raise (libc.so.6)
                                                  #1  0x00007fb9bd984f09 abort (libc.so.6)
                                                  #2  0x00007fb9c0602692 n/a (/home/daenney/.eul/web2/libcef.so)
                                                  #3  0x00000000025ba000 n/a (n/a)
```

Unsure what this is about.

**Comment by voltapp2** [comment]
Hi @daenney 

Sorry for the delay. 

Looks like the built in browser for authentication is crashing. v0.27 will be out soon, it doesn't have CEF, so that will be fixed.

**Comment by daenney** [comment]
Hey no worries. And thanks, that's great news!

**Comment by voltapp2** [comment]
This was fixed in v0.27
